src/nitclk/system_tests/tests_system_nitclk.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+
+import datetime
+import fasteners
+import niscope
+import nitclk
+import os
+import pytest
+import tempfile
+
+
+# nitclk system tests require a different HW that the PXIe-5164 that we simulate else where. It does not support NI-TClk
```

Comment:
I don't think this is accurate. PXIe-5164 does support NI-TClk.
What none of the drivers support is using NI-TClk combined with session-based simulation.

The persistent simulated devices don't need to be DAQmx-framework based devices.